### ISSUE TITLE
prepare datasets

### ISSUE THREAD

**Issue by biexiangduo** [garrickbrazil/M3D-RPN#22]
sorry,i click the "kitti" link.But  there are many links:

----------------------------

    Download left color images of object data set (12 GB)
    Download right color images, if you want to use stereo information (12 GB)
    Download the 3 temporally preceding frames (left color) (36 GB)
    Download the 3 temporally preceding frames (right color) (36 GB)
    Download Velodyne point clouds, if you want to use laser information (29 GB)
    Download camera calibration matrices of object data set (16 MB)
    Download training labels of object data set (5 MB)
    Download object development kit (1 MB) (including 3D object detection and bird's eye view evaluation code)
    Download pre-trained LSVM baseline models (5 MB) used in Joint 3D Estimation of Objects and Scene Layout (NIPS 2011). These models are referred to as LSVM-MDPM-sv (supervised version) and LSVM-MDPM-us (unsupervised version) in the tables below.
    Download reference detections (L-SVM) for training and test set (800 MB)
    Qianli Liao (NYU) has put together code to convert from KITTI to PASCAL VOC file format (documentation included, requires Emacs).
    Karl Rosaen (U.Mich) has released code to convert between KITTI, KITTI tracking, Pascal VOC, Udacity, CrowdAI and AUTTI formats.

--------------------

datasets in this link page  all needed to be downloaded?

**Comment by brazgardian** [comment]
The below data should be downloaded and extracted 
- the left color images
- camera calibration matrices
- training labels

to be more clear

the "data/kitti" folder should look something like this after extraction

training:
data/kitti/training/calib/
data/kitti/training/image_2/
data/kitti/training/label_2/

testing:
data/kitti/testing/calib/
data/kitti/testing/image_2/

Then the validation splits should be setup as described in the readme. Hope this helps!
